### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VarshithRV**
### Why:

Closes [issue link]

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [x] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: data/README.md [hunk 1]
```diff
@@ -39,3 +39,4 @@ Learning tracks are a collection of articles that help you master a particular s
 ## Versioning
 
 Many YAML files in this directory's subdirectories, like [features](features), [glossaries](glossaries), [variables](variables), and [learning tracks](learning-tracks), as well as Markdown files within the [reusables](reusables) directory, can include YAML versioning or Liquid versioning within Markdown strings. See the README.md files in the subdirectories for details.
+harlaharlaharla
```
Comment (ProKashif) [inline data/README.md:42]: Any reason to add this?

**Comment by wantha123456** [comment]
Good

**Comment by wantha123456** [comment]
Good

**Comment by wantha123456** [comment]
Good

**Comment by wantha123456** [comment]
Good

**Pull request closed without merging**